### ISSUE TITLE
Chrome - Which Window Is it?

### ISSUE THREAD

**Issue by Lcstyle** [Morikko/sync-tab-groups#110]
Google Chrome 65.0.3325.181 (not latest, one version behind I think).

In the screen shot below, you can see that sync tab groups main dialog doesn't show which window I am currently on.  I think this is related to the other issue I described at the end of #80 
![si-07-31-52-am_0012](https://user-images.githubusercontent.com/1028272/39243514-afdd37b2-485b-11e8-86dd-0ee7b3148e0e.jpg)

**Comment by Lcstyle** [comment]
OK Here's some additional insight.
Sometimes, sync tab groups does highlight the tab group in the window correctly, I just noticed this behavior.  If I click on another window, and then click back to the "main" window, then tab groups correctly highlights it.   However it depends on WHAT TAB I click on, here's an example:

I click on main window to bring it into focus by clicking on one of the tabs in that window.  Sync Tab Groups is not highlighting the window name in the dialog.

I click away from that window, and click on the first tab in the main window again to bring it back into focus, now magically, the window name is correctly bolded in the tab groups list dialog.

I can reproduce this reliably.

**Comment by Lcstyle** [comment]
yes this issue persists, and i cannot use the extension in google chrome at all because it is mixing up windows, not properly highlighting current window, etc.

**Comment by Lcstyle** [comment]
Here's a error download json from Chrome 

[SyncTabGroups-Log-20180511-210536.zip](https://github.com/Morikko/sync-tab-groups/files/1997152/SyncTabGroups-Log-20180511-210536.zip)

**Comment by Lcstyle** [comment]
Here's a bunch more errors
[SyncTabGroups-Log-20180511-211540.zip](https://github.com/Morikko/sync-tab-groups/files/1997159/SyncTabGroups-Log-20180511-211540.zip)

**Comment by mikhoul** [comment]
There was some promises broken in the latest version for Chrome tomorrow the next version will have fixed the promises so try again **MAYBE** it is the root of the problem especially like me if you use Chrome. 😉

**Comment by Lcstyle** [comment]
Yep,
I can confirm that it's working a lot more reliably now, I ran some quick tests for some behaviors I know would cause the issues, but it seems to be working now.  I will continue testing, but I think you got it.

**Comment by Lcstyle** [comment]
I am now running 0.7 Alpha in Google Chrome:

![si-11-19-27-am_0009](https://user-images.githubusercontent.com/1028272/40006984-b15a6322-5769-11e8-91cf-6d87f2825cad.jpg)

**Comment by Morikko** [comment]
Just some precision: V0.7-alpha is not a stable release by definition. It
might contained new bugs due to the fact I did a lot of change to add the
hiding support. #64
It is available for tests purpose. Be careful with your data.

Else, I will continue to investigate on this problem. Chrome doesn't behave
like Firefox...

**Comment by Lcstyle** [comment]
here's a fairly extensive log set 
[SyncTabGroups-Log-20180514-180632.zip](https://github.com/Morikko/sync-tab-groups/files/2002746/SyncTabGroups-Log-20180514-180632.zip)

**Comment by Morikko** [comment]
I had not the time to do it but I won't forget you.

**Comment by Morikko** [comment]
Just a question: does it still happen ?

**Comment by Lcstyle** [comment]
I just tested it again with 0.7, the behavior now is interesting, I will make a small video to show you how it behaves now

**Comment by Lcstyle** [comment]
![tabgroups chrome 0 7](https://user-images.githubusercontent.com/1028272/42136529-4880635a-7d2b-11e8-9da7-6e1fc24b883d.gif)

**Comment by Lcstyle** [comment]
are you able to reproduce this in chrome on win x64?
Google Chrome is up to date
Version 67.0.3396.99 (Official Build) (64-bit)
